src/services/books/books-service.ts

```diff
@@ -57,13 +60,19 @@ const getBooksByStatus = async (status: string) => {
 };
 
 const createBook = async ({ name, author, year, pages }: BookT) => {
```

Comment:
you don't need de structurization here

Reply:
you mean destructure? Yes, will change this one

Reply:
yes, sorry, I'm not good at English :)